keyboards/neopad/readme.md

```diff
@@ -0,0 +1,23 @@
+# Neopad
+
+Neopad is a little macro pad with four switches and two rotary encoders, made by through hole components only.
+_Actually the number of switches is six, because even the encoder are allowed to click._
+
+![Neopad](https://i.imgur.com/TgOkj2F.jpg "Neopad first proto")
+
+The Neopad in the photo above is the first prototype. See the project repository for revision 1 update and KiCad files.
+
+* Keyboard maintainer: [rookiebwoy](https://github.com/rookiebwoy)
+* Hardware supported: ProMicro, _Elite-C (not tested)_
+* Project repository: [Neopad on github](https://github.com/rookiebwoy/neopad)
+
+Make example for Neopad (after setting up your build environment):
```

Comment:
Please add instructions on how to enter the bootloader to flash new firmware onto the board.

Reply:
Added info to enter booloader.